app/src/lib/utils/post.ts

```diff
@@ -41,6 +41,23 @@ async function handleResponse<T>(response: Response): Promise<T> {
 	if (response.ok) {
 		return response.json();
 	}
+
+	if (response.headers.get('Content-type').includes('application/json')) {
+		const errorMessage = await response.json();
+		return Promise.reject(
+			new HTTPError(errorMessage.message, response.status, response.statusText)
+		);
+	}
```

Comment:
`Promise.reject` est redondant dans une fonction `async` et peut être remplacé par un simple `throw`.

```suggestion
		throw new HTTPError(errorMessage.message, response.status, response.statusText);
```